lightsail.tf

```
@@ -0,0 +1,29 @@
+provider "aws" {
+  version = "~> 2.70.0"
+  region  = var.region
+}
+
+resource "aws_lightsail_key_pair" "key_pair" {
+  name       = var.identifier
+  public_key = var.public_key
+}
+
+resource "aws_lightsail_static_ip" "ip_address" {
+  name = var.identifier
+}
+
+resource "aws_lightsail_instance" "instance" {
+  name              = var.domain_name
+  availability_zone = format("%s%s", var.region, var.availability_zone)
```

We can keep this, but I think the below is nicer, up to you. Could see you not liking mine. Doesn't matter.  I've been wanting to show you this powerful suggestion feature...
```suggestion
  availability_zone = "${var.region}${var.availability_zone}"
```